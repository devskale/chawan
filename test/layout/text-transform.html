<!DOCTYPE html>
<div style="text-transform: capitalize">testing testing testing</div>
<div style="text-transform: full-width">testﾃｽﾄ¥</div>
<div style="text-transform: -cha-half-width">ｔｅｓｔてすとテスト￥</div>
