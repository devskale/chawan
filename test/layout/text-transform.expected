Testing Testing Testing
ｔｅｓｔテスト￥
testﾃｽﾄﾃｽﾄ¥
